main/index.js

```diff
@@ -137,7 +138,12 @@ ipcMain.on('dash:reloadSigner', (e, id) => {
 })
 
 ipcMain.on('tray:openExternal', (e, url) => {
-  if (externalWhitelist.indexOf(url) > -1) shell.openExternal(url)
+  if (externalWhitelist.indexOf(url) > -1) {
```

Comment:
@floating do we still need the first check or does the second check supersede it?